hat/assets/js/apps/Iaso/domains/entities/hooks/requests.ts

```diff
@@ -88,13 +88,14 @@ export const useGetEntitiesApiParams = (
 
 export const useGetEntitiesPaginated = (
     params: Params,
+    isSearchActive: boolean,
```

Comment:
this is ok, but i prefer pass the isEnabled boolean instead, to make it more generic